Copyright.html

```diff
@@ -70,6 +70,8 @@ <h2>Trademarks</h2>
 <p>
 Oracle and Java are registered trademarks of Oracle. Sun, Sun Microsystems, the Java Coffee Cup logo, and the Duke logo are trademarks of Oracle.
 <p>
+CBUS is a registered trademarks of Dr. Michael Bolton. Registered in the United Kingdom.
```

Comment:
Shouldn't the word `trademarks` be singular in this sentence?

Reply:
FIxed